build/push.sh

```diff
@@ -43,7 +43,8 @@ git add CHANGELOG.md dist/cjs/**/* dist/ccxt.bundle.cjs dist/ccxt.browser.js dis
 git add php/test/ php/pro/test/
 git add python/ccxt/test/ python/ccxt/pro/test/
 git add -f python/LICENSE.txt python/package.json python/README.md
-git commit -a -m "${COMMIT_MESSAGE}" -m '[ci skip]' || exit 0
+git commit add .
```

Comment:
Yeah since we add the files "manually" instead of the folders we don't remove old files. I guess we can try to add everything but I think the command is wrong, it should be `git add .` instead of `git commit add  .`  no?

Reply:
you are right
I have fixed it now

Reply:
@olegtaranenko I will merge it soon, but fyi we have a script called `cleanup.sh` that might be useful to clear the repo if you want to change branches

Reply:
one second Carlos. I have a more close look at the script and found that my change denies the previous fine-tuning of the adding specific files

```
git add CHANGELOG.md dist/cjs/**/* dist/ccxt.bundle.cjs dist/ccxt.browser.js dist/ccxt.browser.min.js ts/ccxt.ts ts/src/abstract/*.ts js/**/** php/*.php php/abstract/*.php php/async/*.php php/async/abstract/*.php php/pro/*.php python/ccxt/async_support/*.py python/ccxt/*.py python/ccxt/abstract/*.py python/ccxt/pro/*.py wiki/* examples/py examples/php examples/js
git add php/test/ php/pro/test/
git add python/ccxt/test/ python/ccxt/pro/test/
git add -f python/LICENSE.txt python/package.json python/README.md
```

may be do it smarter? and get rid of not generated exchanges separate?

Reply:
git add -f filename.ext is using, for example, if the filename.ext under .gitignore rules. Why -f flag is using in the snippet? I don't think package.json or readme.md are covered by ignore rules

```     git add -f python/LICENSE.txt python/package.json python/README.md```

Reply:
./cleanup.sh currently left one uncommitted file
```
On branch travis-git-missed-deleted-added-files
Your branch is up to date with 'my/travis-git-missed-deleted-added-files'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        dist/ccxt.browser.min.js.LICENSE.txt

nothing added to commit but untracked files present (use "git add" to track)
```

```
Mac-Pro:ccxt user1$ cat dist/ccxt.browser.min.js.LICENSE.txt 
/*!
Copyright (c) 2011, Yahoo! Inc. All rights reserved.
Code licensed under the BSD License:
http://developer.yahoo.com/yui/license.html
version: 2.9.0
*/

/*! noble-curves - MIT License (c) 2022 Paul Miller (paulmillr.com) */

/*! noble-hashes - MIT License (c) 2022 Paul Miller (paulmillr.com) */

/*! scure-base - MIT License (c) 2022 Paul Miller (paulmillr.com) */
```

looks like an utility file an have to be removed in the generation script

Reply:
> @carlosmiei to me still not clear, why such fine-tuning per-directory practice is used. I've looked at the git history and my impression there is just a historical reason, rather than a great rationale. Travis should never have special 'manually' changed files, which should be avoided to add to the worktree. All artifacts could be added, otherwise, we need to remove them after the generation/transplation steps. Else early or later it produces inconveniences by using git tooling like git bisect or manual branch switching

@olegtaranenko yeah me neither I think I will remove everything and leave only `git add . -A`